examples/ex1.cpp

```diff
@@ -198,8 +199,15 @@ int main(int argc, char *argv[])
    }
    else // Jacobi preconditioning in partial assembly mode
    {
-      OperatorJacobiSmoother M(*a, ess_tdof_list);
-      PCG(*A, M, B, X, 1, 400, 1e-12, 0.0);
+      if (dynamic_cast<const mfem::TensorBasisElement *>(fespace->GetFE(0)))
```

Comment:
This PR has the following more than 5 times:
```
dynamic_cast<const mfem::TensorBasisElement *>(fespace->GetFE(0))
```
This is probably a good time to move it into a reusable method:
```
fespace->usesTensorBasis()
fespace->usesTensorBasis(element)
```